modules/upgrade-scanner-roxctl-371.adoc

```diff
@@ -0,0 +1,45 @@
+
+If you are upgrading to RHACS 3.71 using the `roxctl` CLI and YAML files, you need to perform some additional steps. The Scanner DB image no longer mounts the `scanner-db-password` Kubernetes Secret into the `db` Scanner DB container. Instead, `scanner-db-password` is only used in the init container, `init-db`. Therefore, you must add the `POSTGRES_PASSWORD_FILE` environment variable to the init container configuration. The init container must also mount the `scanner-db-tls-volume` and `scanner-db-password` volumes. The following section provides the upgrade steps for {product-title-short} if you are using {ocp} or Kubernetes. For more information about init containers, see the link:https://kubernetes.io/docs/concepts/workloads/pods/init-containers/[Kubernetes documentation].
+
+.Prerequisites
+
+* This procedure assumes the `db` container in the Scanner DB configuration is at `index 0`, which is the first entry in the `containers` list; and the `scanner-db-password` volume mount is at `index 2`, which is the third entry. 
+
+While this scenario applies to most deployments, check the configuration for Scanner DB before entering these commands. If your values differ, you must adjust the `.../containers/x/volumeMounts/y` value in the following commands. 
+
+.Procedure
+
+. Apply the patch:
+* If you use {ocp}, enter the following command:
++
+[source,terminal]
+----
+$ oc -n stackrox patch deployment.apps/scanner-db --patch '{"spec":{"template":{"spec":{"initContainers":[{"name":"init-db","env":[{"name":"POSTGRES_PASSWORD_FILE","value":"/run/secrets/stackrox.io/secrets/password"}],"command":["/usr/local/bin/docker-entrypoint.sh","postgres","-c","config_file=/etc/postgresql.conf"],"volumeMounts":[{"name":"db-data","mountPath":"/var/lib/postgresql/data"},{"name":"scanner-db-tls-volume","mountPath":"/run/secrets/stackrox.io/certs","readOnly":true},{"name":"scanner-db-password","mountPath":"/run/secrets/stackrox.io/secrets","readOnly":true}],"securityContext":{"runAsGroup":70,"runAsNonRoot":true,"runAsUser":70}}]}}}}'
+----
+* If you use Kubernetes, enter the following command:
++
+[source,terminal]
+----
+$ kubectl -n stackrox patch deployment.apps/scanner-db --patch '{"spec":{"template":{"spec":{"initContainers":[{"name":"init-db","env":[{"name":"POSTGRES_PASSWORD_FILE","value":"/run/secrets/stackrox.io/secrets/password"}],"command":["/usr/local/bin/docker-entrypoint.sh","postgres","-c","config_file=/etc/postgresql.conf"],"volumeMounts":[{"name":"db-data","mountPath":"/var/lib/postgresql/data"},{"name":"scanner-db-tls-volume","mountPath":"/run/secrets/stackrox.io/certs","readOnly":true},{"name":"scanner-db-password","mountPath":"/run/secrets/stackrox.io/secrets","readOnly":true}],"securityContext":{"runAsGroup":70,"runAsNonRoot":true,"runAsUser":70}}]}}}}'
+----
+. Remove the path:
+* If you use {ocp}, enter the following command:
++
+[source,terminal]
+----
+$ oc -n stackrox patch deployment.apps/scanner-db --type json --patch \
```

Comment:
@kcarmichael08 you have a few more instances of odd formatting in the terminal examples for this step. I suggest removing the `\`.